yarn.lock

```diff
@@ -11709,6 +11731,11 @@ typescript@^5.3.3, typescript@^5.4.5:
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-5.4.5.tgz#42ccef2c571fdbd0f6718b1d1f5e6e5ef006f611"
   integrity sha512-vcI4UpRgg81oIRUFwR0WSIHKt11nJ7SAVlYNIu+QpqeyXP+gpQJy/Z4+F0aGxSE4MqwjyXvW/TzgkLAx2AGHwQ==
 
+typescript@next:
+  version "5.6.0-dev.20240613"
```

Comment:
this is odd

Reply:
i just ran `npx yarn-deduplicate && yarn install` and it happened